server/routes/bookedtoday/unexpectedArrivals/validation/dateOfBirthValidation.ts

```diff
@@ -0,0 +1,33 @@
+import moment from 'moment'
```

Comment:
If this is the same validation as currently being used in the `arrivals` folder maybe it's worth moving this up a level so both `arrivals` and `unexpectedArrivals` can use the same validation files? Adding the PNC validation to the existing `validation` folder?

Reply:
it is the same but I think we want to keep things separate

Reply:
ok